### ISSUE TITLE
[QA] DB::Exception: Function with name 'generateSnowflakeID' does not exists. In scope SELECT generateSnowflakeID(1), generateSnowflakeID(2). (UNKNOWN_FUNCTION)

### ISSUE THREAD

**Issue by leiless** [ClickHouse/ClickHouse#65602]
### Question

Hi, all. I'm using ClickHouse server and I want to use the snowflake ID.
But the `SELECT generateSnowflakeID(1), generateSnowflakeID(2);` throws an exception.
Is there any clickhouse setting required to use this functionality?

https://clickhouse.com/docs/en/sql-reference/functions/uuid-functions#generatesnowflakeid

```sql
clickhouse-server :) SELECT VERSION();

SELECT VERSION()

Query id: be9bde66-caba-4677-846d-0927988c38b2

   ┌─VERSION()─┐
1. │ 24.4.3.25 │
   └───────────┘

1 row in set. Elapsed: 0.003 sec.

clickhouse-server :) SELECT generateSnowflakeID(1), generateSnowflakeID(2);

SELECT
    generateSnowflakeID(1),
    generateSnowflakeID(2)

Query id: 351b2c72-3cec-40fd-bce6-cc466e580c8b


Elapsed: 0.004 sec.

Received exception from server (version 24.4.3):
Code: 46. DB::Exception: Received from localhost:9000. DB::Exception: Function with name 'generateSnowflakeID' does not exists. In scope SELECT generateSnowflakeID(1), generateSnowflakeID(2). (UNKNOWN_FUNCTION)
```

**Comment by Algunenano** [comment]
This function is introduced in 24.6, which is not released yet.

**Comment by leiless** [comment]
@Algunenano Just to ensure, is `generateUUIDv7` also introduced in 24.6?

**Comment by Algunenano** [comment]
Yes

**Comment by leiless** [comment]
Is there any cheap way to use `generateSnowflakeID` and `generateUUIDv7` in 23.8-LTS?
Since they are Regular User-Defined Functions. E.g. compile the `generateSnowflakeID` and `generateUUIDv7` UDFs into so files and load in ClickHouse server?

**Comment by Algunenano** [comment]
They are not user defined functions, they are functions included natively in the CH binary.

I guess it might be possible to implement something via pure SQL with bitShift and cast's but maybe it'd be simpler to use some kind of Python UDF. Haven't tried any of them though

**Comment by SergioInToronto** [comment]
Small correction: `generateUUIDv7()` is available on versions before 24.6 (I'm on `24.5.5.78` and it works).

I can confirm `generateSnowflakeID()` works in 24.6
